content/docs/reference/validation_errors@es.md

```diff
@@ -60,7 +60,7 @@ Estas etiquetas obligatorias incluyen un campo `mandatory: true` en la [especifi
 
 El validador emite el error `TAG_REQUIRED_BY_MISSING` cuando encuentra un componente ampliado en el documento de AMP, pero no encuentra su `<script>` equivalente.
 
-Los [componentes ampliados](/docs/reference/extended.html) se deben incluir explícitamente en el documento de AMP como elementos personalizados.
+Los [componentes ampliados](/es/docs/reference/extended.html) se deben incluir explícitamente en el documento de AMP como elementos personalizados.
```

Comment:
This link will break (no redirect).  Change to `es/docs/reference/components.html`